sapi_manifests/cmon/template

```diff
@@ -8,6 +8,7 @@
     "port": 9163,
     "serviceName": "{{SERVICE_NAME}}",
     "serverUuid": "{{auto.SERVER_UUID}}",
+    "discover_include_stopped": "{{discover_include_stopped}}",
```

Comment:
The JSON key name should be camelCase (i.e., `discoverIncludeStopped`). The sapi variable is correct as-is.

Reply:
I'll change it. The existing variable names in the template do not use a consistent naming convention so I wasn't sure which format you guys preferred (eg. `server_opts`, `changefeed_opts`, `throttle_opts`).